prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html

```diff
@@ -201,31 +205,34 @@
         <span>{{ AgreementType[agreement.agreementType] | case: 'snake' : 'space' | capitalize: true }}</span>
       </button>
     </mat-menu>
-
   </ng-template>
 
   <ng-template #siteMenu
                let-site="site"
                let-organizationId="organizationId">
-    <button mat-icon-button
-            [matMenuTriggerFor]="menu"
-            [disabled]="isPendingTransfer()">
-      <mat-icon>more_vert</mat-icon>
-    </button>
-
-    <mat-menu #menu="matMenu"
-              aria-label="Site actions menu">
-      <button mat-menu-item
+    <div class="mat-header-button-group">
+      <button mat-button
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Site Information"
+              [disabled]="isPendingTransfer()"
               (click)="viewSite(organizationId, site)">
-        <span>View/Update Site Information</span>
+        <mat-icon>folder</mat-icon>
+        <span class="pl-1">Site Info</span>
       </button>
-      <button *ngIf="site.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE"
-              mat-menu-item
-              [disabled]="!site.completed || site.status === SiteStatusType.IN_REVIEW"
+      <button mat-button
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Remote Practitioner"
+              *ngIf="site.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE"
+              [disabled]="isPendingTransfer() || !site.completed || site.status === SiteStatusType.IN_REVIEW || site.status === SiteStatusType.LOCKED"
               (click)="viewSiteRemoteUsers(organizationId, site)">
-        <span>View/Update Remote Practitioners</span>
+        <mat-icon>folder_shared</mat-icon>
+        <span class="pl-1">Remote Practitioner</span>
```

Comment:
IMO, this should be "Remote Practitioner**s**"

Reply:
it is updated.